x-pack/plugins/cases/server/client/factory.ts

```diff
@@ -112,6 +115,7 @@ export class CasesClientFactory {
       logger: this.logger,
       authorization: auth,
       auditLogger,
+      actionsClient: await this.options.actionsPluginStart.getActionsClientWithRequest(request),
```

Comment:
Exposing the `actionsClient` through the `CasesClientArgs` so we don't need to access it through the context on each route.